ocs_ci/ocs/elasticsearch.py

```diff
@@ -169,29 +239,18 @@ def cleanup(self):
 
         """
         log.info("Teardown the Elasticsearch environment")
-        log.info(f"Killing the local server process ({self.lspid})")
-        os.kill(self.lspid, signal.SIGKILL)
         log.info("Deleting all resources")
-        subprocess.run(f"oc delete -f {self.crd}", shell=True)
+        log.info("deleting the dumper client pod")
+        self.ocp.delete(yaml_file=self.dumper_file)
+        log.info("deleting the es resource")
+        self.ocp.delete(yaml_file=self.crd)
+        log.info("deleting the es project")
```

Comment:
Same as before - in case you agree, please start from capital D , "Deleting" instead of "deleting"

Reply:
done